src/compiler/transformers/destructuring.ts

```diff
@@ -548,8 +569,13 @@ namespace ts {
                         bindingElements = [];
                     }
                     // Rewrite element to a declaration with an initializer that fetches property
+                    // TODO: Probably save the result of createDestructuringPropertyAccess if propName is a computed property

```

Comment:
Is this TODO something you still have yet to do for this PR or is it for future work?